reference/shaders-no-opt/frag/fp16.invalid.desktop.frag

```diff
@@ -50,7 +50,7 @@ void test_conversions()
     float16_t one = test_result();
     int a = int(one);
     uint b = uint(one);
-    bool c = !(one == float16_t(0.0));
+    bool c = one != float16_t(0.0);
```

Comment:
This file should not change. It failed local testing.

Reply:
Was this commited by mistake?

Reply:
yes, sorry, test suite `--update` did that and i didnt notice